ansible_base/authentication/utils/claims.py

```diff
@@ -424,7 +424,9 @@ def manage_permissions(self) -> None:
         """Processes the user claims (key `rbac_roles`)
         and adds/removes RBAC permissions (a.k.a. role_user_assignments)
         """
-        self.permissions_cache.cache_existing(self.user.role_assignments.all())
+        # NOTE(cutwater): Here `prefetch_related` is used to prevent N+1 problem when accessing `content_object`
```

Comment:
I think `prefetch_related` is always use to prevent N+1, thus the comment is not needed